src/applications/vaos/new-appointment/components/calendar/CalendarWidget.jsx

```diff
@@ -266,23 +263,7 @@ export default function CalendarWidget({
 }) {
   const currentDate = moment();
   const maxMonth = getMaxMonth(maxDate, startMonth);
-  const [months, setMonths] = useState([moment(minDate)]);
-
-  useEffect(
-    () => {
-      // Updates months to show at once if > default setting
-      if (monthsToShowAtOnce > months.length) {
-        const updatedMonths = [];
-        const startDate = startMonth ? moment(startMonth) : moment(minDate);
-
-        for (let index = 0; index < monthsToShowAtOnce; index++) {
-          updatedMonths.push(startDate.clone().add(index, 'months'));
-        }
-        setMonths(updatedMonths);
-      }
-    },
-    [monthsToShowAtOnce, startMonth, months, minDate],
-  );
+  const [months, setMonths] = useState([moment(startMonth || minDate)]);
```

Comment:
This fixes a bug where we weren't starting on the preferred date month anymore.

Reply:
Good catch